Documentation/Development.md

```diff
@@ -134,19 +134,23 @@ $ Utilities/docker-utils run bash # to run an interactive bash shell in the cont
 $ Utilities/docker-utils swift-build # to run swift-build in the container
 $ Utilities/docker-utils swift-test # to run swift-test in the container
 $ Utilities/docker-utils swift-run # to run swift-run in the container
-
 ```
 
-## Using custom compile-only Swift compiler
+## Using custom Swift compilers
+
+SwiftPM needs Swift compiler to parse Package.swift manifest files and to
+compile Swift source files. You can use `SWIFT_EXEC` and `SWIFT_EXEC_MANIFEST`
+environment variables to control which compiler to use for these operations.
 
-You can use a custom Swift compiler for compiling the package sources using the
-environment variable `SWIFT_EXEC_COMPILE_ONLY`. The package manager will use the
-default Swift compiler to parse the manifest files but use the compiler in
-`SWIFT_EXEC_COMPILE_ONLY` to compile the source files. This is mostly useful for
-Swift compiler developers when they want to use a debug compiler with SwiftPM.
+`SWIFT_EXEC_MANIFEST`: This variable controls which compiler to use for parsing
+Package.swift manifest files. The lookup order for manifest compiler is:
+SWIFT_EXEC_MANIFEST, swiftc adjacent to swiftpm binaries, SWIFT_EXEC
```

Comment:
Is the trailing `SWIFT_EXEC` actually right here? From implementation it looks like not `SWIFT_EXEC_MANIFEST ?? resolvedBinDirCompiler`.

Reply:
Yeah, this is correct I thnk. See this if clause https://github.com/apple/swift-package-manager/pull/1444/files#diff-8c6d7a38675f213e4370feadb3a37ed7R87

Reply:
Ah right, I missed that somehow 🙈